Test/FMI3/fmi3_xml_parsing_test.c

```diff
@@ -163,13 +157,10 @@ void test_variable_naming_conventions(void)
 
     /* list of variables */
     fail_name_check("naming_conventions_xmls/list/aemptyc");
-    g_expected_message = "Two variables with the same name";
     pass_name_check("naming_conventions_xmls/list/cba");
-    fail_name_check("naming_conventions_xmls/list/acad");
 
     /* flat hierarchy test */
-    fail_name_check("naming_conventions_xmls/flat/acad");
-    pass_name_check("naming_conventions_xmls/flat/q-char-nonescaped");
+    fail_name_check("naming_conventions_xmls/flat/q-char-nonescaped");
```

Comment:
This one was passing the test incorrectly, since the the expected error messaged being tested was changed in line 166.